terraform-organize-configuration-workshop/step1.md

```diff
@@ -1,169 +1,61 @@
+below.
 
 ```
 provider "aws" {
-  access_key = "REPLACE"
-  secret_key = "REPLACE"
-  region     = var.aws_region
+  region = var.aws_region
 }
 ```{{copy}}
 
-You will need to replace the values for the access_key and secret_key arguments
-with the ones provided to you for this session.
-
-**Warning**: Hard-coding credentials into your Terraform configuration is not
-recommended outside of this lab environment.
-
-## Create Monolithic Configuration
-
-Now that your AWS provider is configured, copy and paste the following example configuration
-into `main.tf`.
+Terraform will use IAM credentials to authenticate with your AWS account. To do
+so, you will need to export two environment variables in the terminal window.
 
 ```
-resource "random_pet" "petname" {
-  length    = 4
-  separator = "-"
-}
-
-resource "aws_s3_bucket" "dev" {
-  bucket = "hc-digital-${var.dev_prefix}-${random_pet.petname.id}"
-  acl    = "public-read"
-
-  policy = <<EOF
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Sid": "PublicReadGetObject",
-            "Effect": "Allow",
-            "Principal": "*",
-            "Action": [
-                "s3:GetObject"
-            ],
-            "Resource": [
-                "arn:aws:s3:::hc-digital-${var.dev_prefix}-${random_pet.petname.id}/*"
-            ]
-        }
-    ]
-}
-EOF
-
-  website {
-    index_document = "index.html"
-    error_document = "error.html"
-
-  }
-  force_destroy = true
-}
-
-resource "aws_s3_bucket_object" "dev" {
-  acl          = "public-read"
-  key          = "index.html"
-  bucket       = aws_s3_bucket.dev.id
-  content      = file("${path.module}/assets/index.html")
-  content_type = "text/html"
-
-}
-
-resource "aws_s3_bucket" "prod" {
-  bucket = "hc-digital-${var.prod_prefix}-${random_pet.petname.id}"
-
-  acl    = "public-read"
-  policy = <<EOF
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Sid": "PublicReadGetObject",
-            "Effect": "Allow",
-            "Principal": "*",
-            "Action": [
-                "s3:GetObject"
-            ],
-            "Resource": [
-                "arn:aws:s3:::hc-digital-${var.prod_prefix}-${random_pet.petname.id}/*"
-            ]
-        }
-    ]
-}
-EOF
-
-  website {
-    index_document = "index.html"
-    error_document = "error.html"
-
-  }
-  force_destroy = true
-}
-
-resource "aws_s3_bucket_object" "prod" {
-  acl          = "public-read"
-  key          = "index.html"
-  bucket       = aws_s3_bucket.prod.id
-  content      = file("${path.module}/assets/index.html")
-  content_type = "text/html"
-}
-```{{copy}}
-
-This configuration includes two s3 buckets set up for static website hosting:
-`dev`, and `prod`.
-
-You may have noticed three variables being used in the above configuration. Define
-these variables by adding the following to `variables.tf`{{open}}.
-
+$ export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
```

Comment:
Break this into separate code blocks? Also it's kind of scary to see example keys in here even if they are expired / dead 😬

Reply:
For the workshop at least, they won't be cutting & pasting these from the katacoda scenario, it will be given to them in Zoom to copy and paste like so:
```
export AWS_ACCESS_KEY_ID=AKIA45W5WTHVWBJIWUMC
export AWS_SECRET_ACCESS_KEY=lBd+8+6hQjcDCWWo1Zd0SFyADp+vFUCqFRAcw4q9
```
For the example values - they aren't actually real keys, I lifted the examples from aws documentation: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html - but I don't really have a preference for which example values we use.